src/main/java/Chris.java

```diff
@@ -0,0 +1,102 @@
+import java.util.Scanner;
+public class Chris {
+    public static void main(String[] args) {
+        String logo = "      ___           ___           ___                       ___     \n" +
+                "     /\\  \\         /\\__\\         /\\  \\          ___        /\\  \\    \n" +
+                "    /::\\  \\       /:/  /        /::\\  \\        /\\  \\      /::\\  \\   \n" +
+                "   /:/\\:\\  \\     /:/__/        /:/\\:\\  \\       \\:\\  \\    /:/\\ \\  \\  \n" +
+                "  /:/  \\:\\  \\   /::\\  \\ ___   /::\\~\\:\\  \\      /::\\__\\  _\\:\\~\\ \\  \\ \n" +
+                " /:/__/ \\:\\__\\ /:/\\:\\  /\\__\\ /:/\\:\\ \\:\\__\\  __/:/\\/__/ /\\ \\:\\ \\ \\__\\\n" +
+                " \\:\\  \\  \\/__/ \\/__\\:\\/:/  / \\/_|::\\/:/  / /\\/:/  /    \\:\\ \\:\\ \\/__/\n" +
+                "  \\:\\  \\            \\::/  /     |:|::/  /  \\::/__/      \\:\\ \\:\\__\\  \n" +
+                "   \\:\\  \\           /:/  /      |:|\\/__/    \\:\\__\\       \\:\\/:/  /  \n" +
+                "    \\:\\__\\         /:/  /       |:|  |       \\/__/        \\::/  /   \n" +
+                "     \\/__/         \\/__/         \\|__|                     \\/__/    ";
+        System.out.println("Hello from\n" + logo );
+
+        Scanner sc = new Scanner(System.in);
+        String command = "";
+        TaskLists listCommands = new TaskLists();
+        System.out.println("____________________________________________________________\n" +
+                "Hello, I'm Chris\n" +
+                "What can I do for you?\n" +
+                "____________________________________________________________\n");
+        while (sc.hasNextLine()) {
+            command = sc.nextLine();
+            if (command.equals("bye")) {
+                System.out.println("____________________________________________________________\n" +
+                        "Bye. Hope to see you again soon!\n" +
+                        "____________________________________________________________");
+                break;
+            } else if (command.equals("list")) {
+                System.out.println("____________________________________________________________");
+                System.out.println("Here are the tasks in your list");
+                listCommands.printTasks();
+                System.out.println("____________________________________________________________");
+            } else if (command.matches("^(mark [0-9]|unmark [0-9])")) {
```

Comment:
I like that you use command as a variable name. Perhaps include a comment to explain how this line works?